### PR TITLE
Bug 1934877: Add configuration for liveness and readiness timeout values for mig registry

### PR DESCRIPTION
**Pull request by pranavgaikwad**

### CODE REVIEW

**Comment by pranavgaikwad** [comment]
Please also review  https://github.com/konveyor/mig-operator/pull/664

File: pkg/apis/migration/v1alpha1/migplan_types.go [hunk 1]
```diff
@@ -47,6 +47,12 @@ const (
 	ClosedIndexField = "closed"
 )
 
+// Registry deployment constants
+const (
+	MigRegistryDefaultLivenessTimeout  = 3
+	MigRegistryDefaultReadinessTimeout = 3
```
Comment (sseago) [inline pkg/apis/migration/v1alpha1/migplan_types.go:53]: The above two values don't seem to be used anywhere. It looks like `RegistryDefaultProbeTimeout` is used instead. Should these be removed?

**Review by sseago — COMMENTED** [review]
Looks good overall, other than the fact that MigRegistryDefaultLivenessTimeout and MigRegistryDefaultReadinessTimeout are defined and never used, while RegistryDefaultProbeTimeout seems to be used instead. We shoud probably either remove them or use them somewhere instead of the general default value.

File: pkg/apis/migration/v1alpha1/migplan_types.go (hunk 1 above)
Comment (pranavgaikwad) [reply pkg/apis/migration/v1alpha1/migplan_types.go:53]: @sseago yeah forgot to remove them. will do now

**Review by sseago — APPROVED** [review]

**Pull request merged**